log.go

```diff
@@ -0,0 +1,31 @@
+package selfupdate
+
+// LogError will be called to log any reason that have prevented an executable update
+var LogError func(string, ...interface{}) = nil
```

Comment:
These ` = nil` should not be needeed, all variables default to the zero value, for a `func` that is `nil`